src/include/uapi/ipc/topology.h

```diff
@@ -113,9 +113,11 @@ struct sof_ipc_comp_config {
 	uint32_t preload_count;	/**< how many periods to preload */
 	uint32_t frame_fmt;		/**< SOF_IPC_FRAME_ */
 	uint32_t xrun_action;
+	uint16_t flavour;
```

Comment:
@lgirdwood Could you proritize merging PRs from @ranj063 once review is done ? it's gating for PR#993 i PR#994

Reply:
I don't want this to be merged without a clearer picture of this notion of flavour. This has not been vetted and agreed on.